runtests.py

```diff
@@ -395,10 +424,12 @@ def get_openmp_compiler_flags(language):
     'tag:openmp': update_openmp_extension,
     'tag:gdb': update_gdb_extension,
     'tag:cpp11': update_cpp11_extension,
+    'tag:cpp17': update_cpp17_extension,
     'tag:trace' : update_linetrace_extension,
     'tag:bytesformat':  exclude_extension_in_pyver((3, 3), (3, 4)),  # no %-bytes formatting
     'tag:no-macos':  exclude_extension_on_platform('darwin'),
     'tag:py3only':  exclude_extension_in_pyver((2, 7)),
+    'tag:execpolicies': require_gcc("9.1")
```

Comment:
I'd call these `cppexecpolicies` or so. `exec` reads too much like `exec()` in Python.